bin/oref0-bluetoothup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if ( ifconfig wlan0 | grep -q "inet addr" ) && ( ifconfig bnep0 | grep -q "inet addr" ); then
+   sudo killall bluetoothd
+fi
+
+if ! ( hciconfig -a | grep -q "PSCAN" ) ; then
+   sudo killall bluetoothd
+   sudo /usr/local/bin/bluetoothd --experimental &
+fi
+
+if ( hciconfig -a | grep -q "DOWN" ) ; then
+   sudo hciconfig hci0 up
+   sudo /usr/local/bin/bluetoothd --experimental &
```

Comment:
I like that, it's much cleaner, though you do have to pass the bt_mac to it, but it's less disruptive to the pancreabble stuff.